src/sql/src/session/vars.rs

```diff
@@ -784,6 +784,37 @@ mod upsert_rocksdb {
                   Only takes effect on source restart (Materialize).",
             internal: true,
         };
+    /// Only used if `upsert_rocksdb_write_buffer_manager_memory_bytes` is also set
+    /// and write buffer manager is enabled
+    pub static UPSERT_ROCKSDB_WRITE_BUFFER_MANAGER_CLUSTER_MEMORY_FRACTION: Lazy<
+        ServerVar<Option<Numeric>>,
+    > = Lazy::new(|| ServerVar {
+        name: UncasedStr::new("upsert_rocksdb_write_buffer_manager_cluster_memory_fraction"),
+        value: &None,
+        description: "Tuning parameter for RocksDB as used in `UPSERT/DEBEZIUM` \
+                  sources. Described in the `mz_rocksdb_types::config` module. \
+                  Only takes effect on source restart (Materialize).",
+        internal: true,
+    });
+    /// `upsert_rocksdb_write_buffer_manager_memory_bytes` needs to be set for write buffer manager to be
+    /// used.
+    pub static UPSERT_ROCKSDB_WRITE_BUFFER_MANAGER_MEMORY_BYTES: ServerVar<Option<usize>> =
+        ServerVar {
+            name: UncasedStr::new("upsert_rocksdb_write_buffer_manager_memory_bytes"),
+            value: &None,
+            description: "Tuning parameter for RocksDB as used in `UPSERT/DEBEZIUM` \
+                  sources. Described in the `mz_rocksdb_types::config` module. \
+                  Only takes effect on source restart (Materialize).",
+            internal: true,
+        };
+    pub static UPSERT_ROCKSDB_WRITE_BUFFER_MANAGER_ALLOW_STALL: ServerVar<bool> = ServerVar {
+        name: UncasedStr::new("upsert_rocksdb_write_buffer_manager_allow_stall"),
+        value: &false,
+        description: "Tuning parameter for RocksDB as used in `UPSERT/DEBEZIUM` \
+                  sources. Described in the `mz_rocksdb_types::config` module. \
+                  Only takes effect on source restart (Materialize).",
+        internal: true,
+    };
```

Comment:
I think this could be a `feature_flags!`. The other not as they're not describing a boolean value.

Reply:
True, since it's a boolean. Ideally feature flags are something we gate features by and then should be removed once it's rolled out? I would lean towards keeping this with the other rocksdb configs. Also, we might change the default of this config to `true` based on more testing.